chaoscenter/web/nginx/nginx.conf

```diff
@@ -63,15 +63,6 @@ http {
         }
 
         location /api/ {
```

Comment:
Please make same change in configmap too in CI manifests - https://github.com/litmuschaos/litmus/tree/master/chaoscenter/manifests

Reply:
Also add required envs (Allowed origins) in CI manifests

Reply:
done